<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="Label_avatar" name="lab_avatar">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QLabel{
border: 2px solid white;
border-radius: 40px;
}
QLabel:hover{
border: 2px solid black;
border-radius: 40px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="LineEdit" name="linee_nickname">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
border: 1px solid rgba(0, 0, 0, 0.3);
border-radius: 10px;
color:grey;
</string>
       </property>
       <property name="text">
        <string>请输入昵称</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="linee_password">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
border: 1px solid rgba(0, 0, 0, 0.3);
border-radius: 10px;
color:grey;
</string>
       </property>
       <property name="text">
        <string>请输入密码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="linee_password_again">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
border: 1px solid rgba(0, 0, 0, 0.3);
border-radius: 10px;
color:grey;
</string>
       </property>
       <property name="text">
        <string>请再次输入密码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>已阅读并同意</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignLeft">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
       font: 9pt &quot;Microsoft YaHei UI&quot;;
       border: none;
       text-decoration: none; 
       color: blue;
   }

   QPushButton:hover {
       text-decoration: underline; 
       color: blue;
   }</string>
         </property>
         <property name="text">
          <string>用户协议</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pu_register">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>37</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font: 12pt 'Microsoft YaHei UI';
    color: white;
    border-radius: 15px;
    background:
        linear-gradient(to bottom right, rgba(255,255,255,0.2), transparent 50%),
        qlineargradient(x1: 1, y1: 0, x2: 1, y2: 1, stop: 0 #c0d0ff, stop: 1 #a0b0e0);
}</string>
       </property>
       <property name="text">
        <string>立即注册</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>83</width>
         <height>90</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>rewritten_widgets/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Label_avatar</class>
   <extends>QLabel</extends>
   <header>rewritten_widgets/label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
